### ISSUE TITLE
Beacon Node powchain error

### ISSUE THREAD

**Issue by rctrinity** [prysmaticlabs/prysm#11200]
Hi,

Trying to get my node up to snuff for upcoming merge and am only receiving this error at the moment.

ERROR powchain: Unable to process past deposit contract logs: no contract code at given address

Is there something I have not yet setup correctly with my GETH (Execution Layer software)? 
My GETH client is currently in healing process, but still fetching new blocks. I haven't setup the API secret yet, but think we are not ready for that yet. I have setup my fee recipient address, and see no issues with that.

Thanks!

**Comment by Piradoxlanieve** [comment]
Do you have the new enpoint configured http://127.0.0.1:8551 ?

**Comment by Piradoxlanieve** [comment]
I did not have the chain of beacons synchronized in goerli, and when updating to prysm 2.1.4, it has stopped synchronizing and gives me an error and restarts,
I have removed the genesis.ssz flag and validator.db no longer crashes but does not start synchronizing the beacon chain

**Comment by rctrinity** [comment]
> Do you have the new enpoint configured http://127.0.0.1:8551 ?

I haven't configured prysm to use 8551 (v2.1.4) as I am running on mainnet, not test. Beacon node is connected to geth through 8545, so no change there yet.  It is syncing and that is not the issue.  Just whines about that error from powchain. Not sure what the contract deposit logs is necessary for powchain, which I assume the error is from GETH.

**Comment by LaplaceMan** [comment]
I also have the same error. Will this error affect the use? How can we solve this problem?

**Comment by prestonvanloon** [comment]
For testnets, your execution engine node needs to be synced to the point at which the deposit contract was deployed. Otherwise the execution engine will fail to serve logs at that contract address because it doesn't exist (yet) in the state of that client. This isn't a problem on mainnet, generally, because the genesis is baked into the client. 

TL;DR -- Sync EE before starting CL.

**Comment by sbakshi007** [comment]
I  am also seeing same error in beacon node traces "ERROR powchain: Unable to process past deposit contract logs, perhaps your execution client is not fully synced error=no contract code at given address".

In geth console, eth.syncing is 'false' - so geth has synced.

curl http://localhost:8080/healthz has following output
*p2p.Service: OK
*execution.Service: ERROR, no contract code at given address
*attestations.Service: OK
*builder.Service: OK
*blockchain.Service: OK
*initialsync.Service: ERROR, syncing
*sync.Service: ERROR, out of sync
*rpc.Service: ERROR, syncing
*gateway.Gateway: OK
*prometheus.Service: OK

And following keep repeating in geth logs..

WARN [11-21|19:11:04.201] Snapshot extension registration failed peer=2e4e37b2 err="peer connected on snap without compatible eth support"
WARN [11-21|19:11:44.083] Served eth_coinbase reqid=3 duration="25.978µs" err="etherbase must be explicitly specified"
WARN [11-21|19:13:42.327] Beacon client online, but never received consensus updates. Please ensure your beacon client is operational to follow the chain!
WARN [11-21|19:13:53.624] Dropping unsynced node during sync id=9ce83f7f0c7d43f5 conn=dyndial addr=168.119.55.61:30303 type=Geth/v1.10.25-stable...

**Comment by james-prysm** [comment]
stale issue, closing.
